esmvalcore/preprocessor/_mask.py

```diff
@@ -226,6 +226,55 @@ def mask_landseaice(cube, fx_files, mask_out):
     return cube
 
 
+def mask_glaciated(cube, mask_out):
+    """
+    Mask out glaciated areas
+
+    It applies a Natural Earth mask. Note that for computational reasons
+    only the 10 largest polygons are used for masking.
+
+    Parameters
+    ----------
+    cube: iris.cube.Cube
+        data cube to be masked.
+
+    mask_out: str
+        "glaciated" to mask out glaciated areas
+
+    Returns
+    -------
+    iris.cube.Cube
+        Returns the masked iris cube.
+
+    Raises
+    ------
+    ValueError
+        Error raised if masking on irregular grids is attempted.
+        Irregular grids are not currently supported for masking
+        with Natural Earth shapefile masks.
+    """
+
+    # Dict to store the Natural Earth masks
+    cwd = os.path.dirname(__file__)
+    # read glaciated shapefile
+    shapefiles = {
+        'glaciated': os.path.join(cwd, 'ne_masks/ne_10m_glaciated_areas.shp'),
+    }
+    if mask_out == 'glaciated':
+        cube = _mask_with_shp(cube, shapefiles[mask_out],
+                              [1859, 1860, 1861, 1857, 1858,
+                               1716, 1587, 1662, 1578, 1606, ]
+                             )
```

Comment:
are those indices of the geometries or years? Hope you didnt have to go through 1850 previous indices :grin:

Reply:
also pep is complaining about the closing bracket

Reply:
> are those indices of the geometries or years? Hope you didnt have to go through 1850 previous indices

Haha, no. They are the indices corresponding to the largest polygons. :)

> also pep is complaining about the closing bracket
For me pep doesn't complain...  ? I check it by running `prospector`.

Reply:
I found the complaint, it was flake8, should be solved now.